Git-Closure/English.md

```diff
@@ -0,0 +1,168 @@
+# New feature in Git 3: closures
+
+Git is a popular version control system. In Git, an atomic change of one or
+several files is called a commit, and several consecutive commits are combined
+into a branch. Branches are used to implement new ideas (features).
+
+![Git Logo](Logo.svg)
+
+It happens that the idea is a dead end and the developer has turned the wrong
+way, so he needs to roll back to the original version. He should forget about
+the new branch, switch to the main **dev** or **master** branch and continue
+working. In this case, the "scion" will hang forever, as well as the desire to
+remove it. But how to remove the part of the history? This branch shows the
+efforts of the hard-working programmer, even if in vain. So it will be easier to
+report to the boss, because an unsuccessful result is also a result!
+
+I hasten to rejoice that Git developers are going to introduce a new command to
+close such "homeless" branches in the third version. The current version is
+[2.21.0](https://git-scm.com/downloads).
+
+How to use this command, what benefits does it give and what do IT companies
+think? The article answers these and other questions.
+
+<cut/>
+
+## Description
+
+Now it is possible to close an unsuccessful branch over one of the previous
+commits. Closure arcs are yellow colored in the pictures below.
```

Comment:
I'd suggest "Closure arcs are painted in yellow in the pictures below." or something along these lines.